.changeset/early-pillows-jog.md

```diff
@@ -0,0 +1,7 @@
+---
+"@builder.io/sdk-react-native": major
```

Comment:
Not major, that's reserved for big breaking changes. You'll want to go with a patch change here:

```suggestion
"@builder.io/sdk-react-native": patch
```